model-server-openapi/specifications/model-server-operative.yaml

```diff
@@ -15,18 +15,56 @@ info:
   version: "1.0.0"
 
 servers:
-  - url: '/'
+  - url: '/meta'
```

Comment:
`/meta` is very ambiguous in our context. Maybe would should use something linke `administration`/`introspection`/`control`/`server`.